### PR TITLE
 [12.0][MIG] shopinvader_locomotive

### PR DESCRIPTION
**Pull request by rousseldenis**
Includes :

* [x] #320 

Depends on:

* [ ] https://github.com/akretion/locomotivecms-python-client/pull/1

IMPORTANT: We removed dependency on keychain as the module seems to be deprecated in v12 : https://github.com/OCA/server-auth/pull/39

This allows to be free on the implementation on how to store identification data (and to implement a specific one on top of this).

### CODE REVIEW

File: shopinvader_locomotive/data/ir_cron.xml [hunk 1]
```diff
@@ -9,9 +9,8 @@
...
         <field name="numbercall">-1</field>
         <field eval="False" name="doall"/>
-        <field eval="'shopinvader.backend'" name="model"/>
-        <field eval="'_scheduler_synchronize_currency'" name="function"/>
-        <field eval="'()'" name="args"/>
+        <field name="model_id" ref="model_shopinvader_backend"/>
+        <field name="code">model._scheduler_synchronize_currency()</field>
     </record>
 
     <record forcecreate="True" id="ir_cron_export_binding" model="ir.cron">
```
Comment (simahawk) [inline shopinvader_locomotive/data/ir_cron.xml:16]: I don't get why this is needed. Even if needed, shouldn't be placed into shopinvader main module or shopinvader_search_engine?

**Review by simahawk — APPROVED** [review]
LG. 2 things:
1. I don't get why indexes export should be triggered every day here (see comment)
2. would be nice to fast track this so that I can split the generic parts to `shopinvader_client_base` or similar ;)

File: shopinvader_locomotive/data/ir_cron.xml (hunk 1 above)
Comment (lmignon) [reply shopinvader_locomotive/data/ir_cron.xml:16]: @simahawk, you're right. This one should go into shopinvader_search_engine.

**Comment by lmignon** [comment]
>     2. would be nice to fast track this so that I can split the generic parts to `shopinvader_client_base` or similar ;)

I'll fix the build and merge once travis is :green_apple:

**Comment by lmignon** [comment]
@simahawk It's merged!

**Pull request merged**